tests/unit/ConsentContinuationOut.spec.ts

```diff
@@ -153,29 +149,29 @@ describe('Consent to Continuation Out view', () => {
   })
 
   it('saves draft consent to continuation out properly', async () => {
-    // mock "save draft" endpoint (garbage response data - we aren't testing that)
-    sinon.stub(axios, 'post')
-      .withArgs('businesses/CP1234567/filings?draft=true')
-      .returns(new Promise(resolve =>
-        resolve({
-          data: {
-            filing: {
-              business: {},
-              header: { filingId: 456 },
-              consentContinuationOut: { details: 'test' },
-              annualReport: {}
-            }
-          }
-        })
-      ))
+    // mock "has pending tasks" legal service
+    jest.spyOn(LegalServices, 'hasPendingTasks').mockImplementation((): any => {
+      return Promise.resolve(false)
+    })
+
+    // mock "create filing" legal service
+    // (garbage response data - we aren't testing that)
+    jest.spyOn(LegalServices, 'createFiling').mockImplementation((): any => {
+      return Promise.resolve({
+        business: {},
+        header: { filingId: 456 },
+        consentContinuationOut: { details: 'test' },
+        annualReport: {}
+      })
+    })
 
     // mock $route
-    const $route = { params: { filingId: '456' } }
+    const $route = { params: { filingId: '0' } }
```

Comment:
This was another bug: as we are saving a new filing, the filing ID was expected to be 0.

Because this wasn't zero, some code was bypassed.